home.css

```diff
@@ -0,0 +1,173 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700;800;900&display=swap');
+
+body {
+  font-family: 'Inter', sans-serif;
+  margin: 0;
+  padding: 0;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p,
+pre {
+  padding: 0;
+  margin: 0;
+}
+.nav-bar {
+  top: 0;
+  position: fixed;
+  width: 100%;
+  background-color: rgb(236, 231, 231);
+  padding: 1rem 2rem;
+}
+
+.main-div {
+  margin-top: 5rem;
+  display: flex;
+  flex-direction: row;
+}
+
+.side-bar {
+  overflow-y: auto;
+
+  display: flex;
+  flex-direction: column;
+  background-color: white;
+  width: 20%;
+  height: calc(100vh - 5rem);
+}
+
+.nav-item {
+  margin: 0.5rem 1rem;
+  font-weight: 500;
+  color: #2d3748;
+  font-size: 14px;
+  line-height: 24px;
+}
+.mid-div {
+  width: 80%;
+  padding: 0 10rem;
+  overflow-y: auto;
+  height: calc(100vh - 5rem);
+}
+.nav-item a {
+  text-decoration: unset;
+  color: #2d3748;
+}
+.nav-item > * {
+  cursor: pointer;
+}
+.nav-items-heading {
+  font-weight: 700;
+}
+
+.nav-sub-items {
+  display: flex;
+  flex-direction: column;
+  margin-left: 0.5rem;
+}
+/* Avatar */
+.avatar-container {
+  margin: 1rem 0;
+  display: flex;
+  flex-direction: row;
+  gap: 1rem;
+}
+.avatar {
+  background-color: grey;
+  width: 50px;
+  height: 50px;
+  border-radius: 50%;
+  overflow: hidden;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+.avatar img {
+  height: 100%;
+  width: 100%;
+}
+
+.box-xs {
+  width: var(--xs);
+  height: var(--xs);
+}
+.box-s {
+  width: var(--s);
+  height: var(--s);
+}
+.box-m {
+  width: var(--m);
+  height: var(--m);
+  font-size: var(--xs);
+}
+.box-l {
+  width: var(--l);
+  height: var(--l);
+}
+.box-xl {
+  width: var(--xl);
+  height: var(--xl);
+}
+.box-xxl {
+  width: var(--xxl);
+  height: var(--xxl);
+}
+.avatar-code {
+  background-color: rgb(1, 22, 39);
+  height: 100px;
+  overflow-y: auto;
+  color: white;
+}
+
+/* Alert */
+
+.alert {
+  border-radius: var(--xs);
+  margin-bottom: var(--xl);
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  padding: var(--s);
+}
+.alert-icon {
+  font-size: var(--m);
+  margin-right: var(--s);
+}
+.alert-content {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
```

Comment:
you can create a utility class for these properties as they will occur very often